.github/workflows/cryostat-IT.yml

```diff
@@ -0,0 +1,119 @@
+on:
+  workflow_call:
+    inputs:
+      build-arch:
+        required: true
+        type: string
+      skip-itests:
+        required: false
+        type: boolean
+        default: false
+    outputs:
+      image-version:
+        description: the Cryostat application version that will be built
+        value: ${{ jobs.get-pom-properties.outputs.image-version }}
+
+permissions:
+  contents: read
+  packages: read
+
+jobs:
+  get-pom-properties:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: cryostatio/cryostat
+        ref: main
+    - id: query-pom
+      name: Get properties from POM
+      # Query POM for image version and save as output parameter
+      run: |
+        IMAGE_VERSION="$(mvn validate help:evaluate -Dexpression=cryostat.imageVersionLower -q -DforceStdout)"
+        echo "image-version=$IMAGE_VERSION" >> "$GITHUB_OUTPUT"
+    outputs:
+      image-version: ${{ steps.query-pom.outputs.image-version }}
+
+  build-image:
+    runs-on: ubuntu-latest
+    needs: [get-pom-properties]
+    steps:
+    - name: Install qemu
+      if: ${{ inputs.build-arch != 'amd64' }}
+      continue-on-error: true
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y qemu-user-static
+    - uses: actions/checkout@v4
+      if: always()
+      with:
+        repository: cryostatio/cryostat
```

Comment:
```yaml
with:
  owner: ${{ github.repository_owner }}
  name: cryostat
  ...
```

We should use the owner annotation here?

Reply:
And else where.

Reply:
there's no option for an "owner" for actions/checkout@v4

Reply:
using `repository: ${{ github.repository_owner }}/cryostat` instead